src/codex_query.py

```diff
@@ -140,68 +127,62 @@ def get_query(prompt_file):
 
     # get input from terminal or stdin
     if DEBUG_MODE:
-        entry = input("prompt: ") + '\n'
+        entry = input("prompt: ")
     else:
-        entry = sys.stdin.read()
+        # Remove extreaneous newlines and hashtag from the input string
+        entry = sys.stdin.read().strip("\n").strip("#").strip(" ") 
     # first we check if the input is a command
-    command_result, prompt_file = get_command_result(entry, prompt_file)
+    try:
+        command_result, prompt_generator = get_command_result(entry, prompt_generator)
+    except Exception as e:
+        print (str(e))
 
     # if input is not a command, then query Codex, otherwise exit command has been run successfully
```

Comment:
could you revisit this comment, I'm not sure it reflects the logic below, it's quite hard to understand the rationale deciding when to exit - thanks